be/src/olap/rowset/segment_v2/segment_writer.cpp

```diff
@@ -475,16 +475,15 @@ Status SegmentWriter::probe_key_for_mow(
                                       specified_rowsets, &loc, _mow_context->max_version,
                                       segment_caches, &rowset);
     if (st.is<KEY_NOT_FOUND>()) {
-        if (_opts.rowset_ctx->partial_update_info->is_strict_mode) {
-            ++stats.num_rows_filtered;
+        auto ignore_cb = [&]() {
```

Comment:
The `ignore` operation will not work after compaction, compaction will ignore delete bitmap, and keep the latest key, unless you add a delete sign to such row.
But adding delete sign might have some issue if user specified sequence column.

Reply:
Maybe we need to consider to remove IGNORE option
The IGNORE option works exactly like the reverted PR for `INSERT IGNORE`